zabdig

```diff
@@ -669,9 +687,15 @@ class ZInterface(object):
 
             # zabbix-server >= 6.0 does not have a proxy.name anymore. Use the
             # host instead.
+            host_label = 'name'
+            address_label = 'address'
+            if self.version < (7,):
+                host_label = 'host'
+                address_label = 'proxy_address'
+
             proxies = self._zabbix_call(
                 'proxy.get',
-                {'output': ['host', 'proxy_address', 'proxyid']})
+                {'output': [host_label, address_label, 'proxyid']})
```

Comment:
Does this not need a compat_replacement then?

Reply:
It actually wasn't needed since that case was handled elsewhere, but I've added the `compat_replacement` and removed the bit that was handling this case before.

```python
try:
    self.name = hostinfo['name']
except KeyError:
    assert 'proxy_address' in hostinfo, hostinfo
    self.name = hostinfo['host']  # proxies do not have a name
```